src/html.js

```diff
@@ -1,6 +1,9 @@
 import React from "react"
 import PropTypes from "prop-types"
 
+let twitter = "!function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);},s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');twq('init','o65mg');twq('track','PageView');";
+let mixpnljs = "var _mtm = window._mtm = window._mtm || [];_mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];g.type='text/javascript'; g.async=true; g.src='https://cdn.matomo.cloud/leanmarketing.matomo.cloud/container_zuHO1C32.js'; s.parentNode.insertBefore(g,s);";
```

Comment:
@rootedbox this is not mixpanel snippet. This is Matomo Cloud. Please add the mixpanel specific tracking to the site/blog where it may be missing. It may not be missing.